proposals/primer/application.bs

```diff
@@ -0,0 +1,353 @@
+<pre class='metadata'>
+Title: Solid Application Interoperability - Application Primer
+Shortname: solid-interop-application-primer
+Level: 1
+Status: w3c/ED
+Group: solid-cg
+URL: https://solid.github.io/data-interoperability-panel/primer/applications.html
+Repository: https://github.com/solid/data-interoperability-panel
+Inline Github Issues: title
+Editor: elf Pavlik
+Abstract: Primer for libraries intented to be used by applications.
+Boilerplate: omit conformance
+</pre>
+
+# Introduction # {#introduction}
+
+This primer was developed along side open source TypeScript implementation [[SAI-JS]]
+
+# Shape Trees # {#shape-trees}
+
+Shape Trees [[SHAPETREES]] allow creating virtual data hierarchies and define which shapes [[SHEX]]
+should be used to validate data. In our examples we will use following Shapes and Shape Trees.
```

Comment:
"...allow for the definition and validation of physical and/or virtual data hierarchies including which shapes..."